types/ember-test-helpers/index.d.ts

```diff
@@ -17,10 +17,10 @@ declare module 'ember-test-helpers' {
         unit?: boolean;
         needs?: string[];
 
-        beforeSetup?(): void;
-        setup?(): void;
-        teardown?(): void;
-        afterTeardown?(): void;
+        beforeSetup?(assert?: any): void;
```

Comment:
I assume `assert` here is the type of `QUnit.assert`?

Reply:
The assert argument is there for qunit and not used by mocha .I think ember-test-helpers treats it like an opaque value. I override the interface to use `qunit.Assert` in the qunit typings.

Reply:
Ah, that makes sense. 👍